sqlmesh/core/console.py

```diff
@@ -33,8 +33,8 @@
     SnapshotChangeCategory.BREAKING: "Breaking",
     SnapshotChangeCategory.NON_BREAKING: "Non-breaking",
     SnapshotChangeCategory.FORWARD_ONLY: "Forward-only",
-    SnapshotChangeCategory.INDIRECT_BREAKING: "Indirect breaking",
-    SnapshotChangeCategory.INDIRECT_FORWARD_ONLY: "Indirect forward-only",
```

Comment:
why did this change? was there a discussion around this?

Reply:
I talked to @izeigerman and I said I found `Indirect Forward-only` confusing since it by name seems to imply children of a forward-only change when it is really children of a non-breaking change.